packages/client/src/exchanges/types/exchangeFilters.types.ts

```diff
@@ -0,0 +1,45 @@
+import type { Order, OrderItem } from '../../orders/index.js';
+
+export type ExchangeFilters = {
+  id: string;
+  exchangeFilterItems: Array<ExchangeFilterItem>;
+  logicOperator: LogicOperator;
+};
+
+export type ExchangeFilterItem = {
+  orderCode: Order['id'];
+  orderItemUuid: OrderItem['shippingOrderLineId'];
+};
+
+export type LogicOperator = {
+  logicOperatorType: LogicOperatorType;
+  operators: Array<Operators>;
+};
+
+export enum LogicOperatorType {
+  None = 'None',
+  And = 'And',
+  Or = 'Or',
+}
+
+export type Operators = {
```

Comment:
```suggestion
export type LogicOperatorData = {
```

Reply:
Actually scratch that, use this instead:

```suggestion
export type ExchangeFilterLogicOperatorData = {
```

Reply:
This should apply also to `LogicOperatorType` which should be renamed to `ExchangeFilterLogicOperatorType`

Reply:
Forgot this one. Done 👍